src/gui/Src/Gui/CPUDisassembly.cpp

```diff
@@ -1556,6 +1558,25 @@ void CPUDisassembly::copyAddressSlot()
     Bridge::CopyToClipboard(clipboard);
 }
 
+void CPUDisassembly::copyRawAddressSlot()

```

Comment:
What exactly does this copy? The byte pattern `AA BB CC DD` or am I misunderstanding?

![image](https://user-images.githubusercontent.com/2458265/167168921-b8fa0832-7bf1-4146-be84-9f9d63ec7c7c.png)

If I understand correctly this should do the same.

Reply:
Hi, go in the "Copy" section and look for "Address to pattern". This option will just take the memory address formatted as a raw pattern, ready to be written (pasted) in memory resulting as a valid pointer without having to manually decompose the address.